hummingbot/strategy/pure_market_making/constant_spread_pricing_delegate.pyx

```diff
@@ -27,13 +27,12 @@ cdef class ConstantSpreadPricingDelegate(OrderPricingDelegate):
                                            list active_orders):
         cdef:
             MarketBase maker_market = market_info.market
-            OrderBook maker_order_book = maker_market.c_get_order_book(market_info.trading_pair)
-            object top_bid_price = Decimal(maker_order_book.c_get_price(False))
-            object top_ask_price = Decimal(maker_order_book.c_get_price(True))
+            object top_bid_price = market_info.get_price(False)
+            object top_ask_price = market_info.get_price(True)
             str market_name = maker_market.name
-            object mid_price = (top_bid_price + top_ask_price) * Decimal(0.5)
-            object bid_price = mid_price * Decimal(1.0 - self._bid_spread)
-            object ask_price = mid_price * Decimal(1.0 + self._ask_spread)
+            object mid_price = (top_bid_price + top_ask_price) * Decimal("0.5")
+            object bid_price = mid_price * (1 - self._bid_spread)
```

Comment:
`Decimal(1)`

Yes the results will be the same, due to automatic type casting, and that 1 can be perfectly represented in IEEE754... but it'd be more consistent.